Kernel/KerrGeoOrbit.m

```diff
@@ -26,6 +26,9 @@
 KerrGeoOrbit::OutOfBounds = "For this hyperbolic orbit the Darwin parameter \[Chi] must be between `1` and `2`"
 
 
+KerrGeoOrbit::OnSeparatrix = "This orbit is on the separatrix where many expressions are numerically singular. Aborting..."
+
+
 (* ::Subsection::Closed:: *)
```

Comment:
It would be better if we could fix the KerrGeoMinoFrequency* functions to work in this case.

Reply:
I agree, but from my preliminary look at this, there doesn't seem to be an easy finite limit that can be taken. These frequencies may need to be completely re-derived for this case. Hence the next best thing of stopping the user from using the functions on the separatrix until these are known.

Reply:
Are there common sub-cases (e.g. ISCO) where we could make it work, but leave the generic case for another time?